webpack.mix.js

```diff
@@ -22,8 +22,24 @@ const path = require('path');
 const webpack = require('webpack');
 const SentryPlugin = require('webpack-sentry-plugin');
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+
+// might have already been called when mix starts? Won't override anything.
```

Comment:
maybe just

```
dotenv.config({path: `${__dirname}/.env`});
```

Reply:
yarn or webpack seems smart enough to run with the correct cwd, and doesn't look like we need to run this anymore since it's called by mix right when it starts, so it doesn't actually do anything anymore 🥉